drivers/fpga/fpga_altera.c

```diff
@@ -0,0 +1,463 @@
+/*
```

Comment:
File name should have agilex bridge appended

Reply:
done